src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int initialCapacity = 16;
+    private float loadFactor = 0.75f;
+    private int size = 0;
+    private LinkedList<Entry<K, V>>[] buckets;
+    private Entry<K, V> nullKeyEntry;
+
+    public MyHashMap() {
+        buckets = new LinkedList[initialCapacity];
+    }
+
+    private static class Entry<K, V> {
+        private K key;
+        private V value;
+
+        Entry(K key, V value) {
```

Comment:
Don't begin method implementation with an empty line. Please remove the empty line at the start of the `Entry` constructor.